src/components/CategoryFilter.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CategoryFilter = ({ handleFilter }) => {
+  const categories = ['All', 'Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+  return (
+    <>
+      <label>
+        Category Filter
+        <select onChange={handleFilter}>
+          {categories.map((cat, index) => (
+            <option key={index} value={cat}>{cat}</option>
```

Comment:
- [ ] It's bad practice to use the index of the array as a key. You can for this use case make use of the values of the array, since they are unique.
- [ ] By making this above fix then you can remove the eslint disabled rule at the top of the module.